src/ansys/fluent/core/workflow.py

```diff
@@ -925,20 +925,20 @@ def __call__(self):
         """Delegate calls to the underlying workflow."""
         return self._workflow()
 
-    def _wait_on_refresh(self, time_unit=0.1, skip_check=False):
-        if not skip_check:
-            t0 = time()
-        if skip_check or threading.get_ident() == self._main_thread_ident:
-            refresh_count = self._refresh_count
-            orig_refresh_count = refresh_count
-            sleep(20 * time_unit)
-            while self._refreshing or self._refresh_count > refresh_count:
-                refresh_count = self._refresh_count
-                sleep(time_unit)
-            if self._refresh_count > orig_refresh_count:
-                self._wait_on_refresh(time_unit=time_unit, skip_check=True)
-        if not skip_check:
-            logger.debug("_wait_on_refresh time taken {time() - t0}")
+    # def _wait_on_refresh(self, time_unit=0.1, skip_check=False):
```

Comment:
@prmukherj   Just noticing other sleep calls. Please see ln 986, and see if we need those.

Reply:
I have noticed that already. This sleep can be removed as well but it will do this step repeatedly: logger.debug("Call _refresh_task_accessors"). So keeping the small sleep here might be useful, but it can be removed otherwise. What do you think @seanpearsonuk ?

Reply:
I would hope that we can access all the task information immediately after creating the workflow. So, please let's try it. If it doesn't work then we will need a better architectural solution.

Reply:
ok